lld/test/ELF/aarch64-patchinst.s

```diff
@@ -0,0 +1,87 @@
+# RUN: rm -rf %t && split-file %s %t
```

Comment:
consider adding `&& cd %t` so that we can remove `%t/` below, which clutter up the commands...

Reply:
I prefer this form so that the commands are copy-pastable into the shell.